tests/foreman/cli/test_rhcloud_inventory.py

```diff
@@ -51,8 +55,30 @@ def test_positive_inventory_generate_upload_cli():
 
     :expectedresults: Inventory is generated and uploaded to cloud.redhat.com.
 
-    :CaseAutomation: NotAutomated
+    :CaseAutomation: Automated
 
     :CaseLevel: System
     """
-    pass
+    org, ak = organization_ak_setup
+    cmd = f'organization_id={org.id} foreman-rake rh_cloud_inventory:report:generate_upload'
+    upload_success_msg = f'Generated and uploaded inventory report for organization \'{org.name}\''
```

Comment:
```suggestion
    upload_success_msg = f'Generated and uploaded inventory report for organization "{org.name}"'
```

Reply:
Assertion won't work if I do this. success message contains organization name in single quote(') and not double quotes("). That's why I have written `upload_success_msg` like that.